src/pages/admin/tokens/edit.tsx

```diff
@@ -142,33 +159,45 @@ async function assembleAuctionHouses(
     const [auctionHouse, bump] =
       await AuctionHouseProgram.findAuctionHouseAddress(publicKey, tMintKey)
 
-    const [feeAccount, feePayerBump] =
-      await AuctionHouseProgram.findAuctionHouseFeeAddress(auctionHouse)
-
-    const [treasuryAccount, treasuryBump] =
-      await AuctionHouseProgram.findAuctionHouseTreasuryAddress(auctionHouse)
-
-    const auctionHouseCreateInstruction = createCreateAuctionHouseInstruction(
-      {
-        treasuryMint: tMintKey,
-        payer: wallet.publicKey,
-        authority: wallet.publicKey,
-        feeWithdrawalDestination: fwdKey,
-        treasuryWithdrawalDestination: twdAta,
-        treasuryWithdrawalDestinationOwner: twdKey,
-        auctionHouse,
-        auctionHouseFeeAccount: feeAccount,
-        auctionHouseTreasury: treasuryAccount,
+    // Check if auctionHouse already exists, if exist dont add createInstruction, just return auctionHouse
+    const ahQuery = useQuery<GetAuctionHouse>(GET_AUCTION_HOUSE, {
+      variables: {
+        address: auctionHouse.toBase58(),
```

Comment:
hooks can only be used within a render function. For this you'll want to get the apolo client.